doc/en/user/source/services/wfs/vendor.rst

```diff
@@ -106,3 +64,50 @@ Although providing an invalid namespace will not result in any errors, the GetCa
 
 .. note:: This filter is related to :ref:`virtual_services`.
 
+GetFeature Request
+------------------
+
+CQL filters
+^^^^^^^^^^^
+
+In WFS :ref:`wfs_getfeature` GET requests, the ``cql_filter`` parameter can be used to specify a filter in ECQL (Extended Common Query Language) format. ECQL provides a more compact and readable syntax compared to OGC XML filters. 
+
+For full details see the :ref:`filter_ecql_reference` and :ref:`cql_tutorial` tutorial.
+
+The following example illustrates a GET request OGC filter:
+
+:: 
+
+   filter=%3CFilter%20xmlns:gml=%22http://www.opengis.net/gml%22%3E%3CIntersects%3E%3CPropertyName%3Ethe_geom%3C/PropertyName%3E%3Cgml:Point%20srsName=%224326%22%3E%3Cgml:coordinates%3E-74.817265,40.5296504%3C/gml:coordinates%3E%3C/gml:Point%3E%3C/Intersects%3E%3C/Filter%3E
+
+Using ECQL, the identical filter would be defined as follows:
+
+::
+
+   cql_filter=INTERSECTS(the_geom,%20POINT%20(-74.817265%2040.5296504))
+
+
+Format options
+^^^^^^^^^^^^^^
+
+The ``format_options`` parameter is a container for other parameters that are format-specific. The syntax is::
+  
+    format_options=param1:value1;param2:value2;...
+    
+The supported format option is:
+
+* ``callback`` (default is ``parseResponse``)—Specifies the callback function name for the JSONP response format
+* ``id_policy`` (default is ``true``)- Specifies id generation for the JSON output format. To include feature id in output use an attribute name, or use ``format_options=id_policy:true`` for feature id generation. To avoid the use of feature id completely use ``format_options=id_policy:false``.
```

Comment:
```suggestion
* ``id_policy`` (default is ``true``) - Specifies id generation for the JSON output format. To include feature id in the output, use an attribute name, or use ``format_options=id_policy:true`` for feature id generation. To avoid the use of feature id completely use ``format_options=id_policy:false``.
```